src/main/java/org/skriptlang/skript/bukkit/breeding/elements/CondCanAge.java

```diff
@@ -16,20 +22,38 @@
 		"\tbroadcast \"An immortal has been born!\" to player"
 })
 @Since("2.10")
-public class CondCanAge extends PropertyCondition<LivingEntity> {
+public class CondCanAge extends PropertyCondition<LivingEntity> implements SyntaxRuntimeErrorProducer {
 
-	static {
-		register(CondCanAge.class, PropertyType.CAN, "(age|grow (up|old[er]))", "livingentities");
+	public static void register(SyntaxRegistry registry) {
+		register(registry, CondCanAge.class, PropertyType.CAN, "(age|grow (up|old[er]))", "livingentities");
+	}
+
+	private Node node;
+
+	@Override
+	public boolean init(Expression<?>[] expressions, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		node = getParser().getNode();
+		return super.init(expressions, matchedPattern, isDelayed, parseResult);
 	}
 
 	@Override
 	public boolean check(LivingEntity entity) {
-		return entity instanceof Breedable breedable && !breedable.getAgeLock();
+		if (entity instanceof Breedable breedable) {
+			return !breedable.getAgeLock();
+		} else {
+			warning("An entity passed through wasn't breedable, and thus returned false by default.");
+			return false;
+		}
```

Comment:
this doesn't seem like it needs a warning. If it's not breedable, it can't grow up.